macros/CurrentGecko.ejs

```diff
@@ -55,12 +54,6 @@ var versionInfo = mdn.fetchJSONResource(url);
 
 var version = versionInfo[versionID];
 
-if (bumpToNightly) {
-    var major = parseInt(version, 10);  // Get major version number
-    major++;
-    version = major + version.slice(major.toString().length);
```

Comment:
**Fact:** This had a subtle bug where the original version was `9` (or `99`), which would cause `major++` to increment it to `10` (or `100`), which would cause `major.toString().length` to return a value `1` higher than what was intended, which would cause the `version` string to contain an incorrect value.

Glad that that’s been fixed.